src/scenes/home/benefit/benefit.js

```diff
@@ -155,27 +172,66 @@ const Benefit = () => (
             target="_blank"
             rel="noopener noreferrer"
           >
-            <img className={styles.sabioLogo} src={Sabio} alt="Sabio Logo" />
+            <img className={styles.logo} src={Sabio} alt="Sabio" />
           </a>
           <a
             href="https://www.nodesummit.com"
             target="_blank"
             rel="noopener noreferrer"
           >
-            <img className={styles.nodeLogo} src={Nodejs} alt="Node Foundation Logo" />
+            <img className={styles.logo} src={Nodejs} alt="Node Summit" />
           </a>
           <a
             href="https://www.codefellows.org/"
             target="_blank"
             rel="noopener noreferrer"
           >
-            <img className={styles.codeFellowsLogo} src={CodeFellows} alt="Code Fellows" />
+            <img className={styles.logo} src={CodeFellows} alt="Code Fellows" />
+          </a>
+          <a
+            href="https://secureset.com/"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <img className={styles.logo} src={secureSet} alt="Secure Set Cybersecurity Bootcamp" />
+          </a>
+          <a
+            href="https://www.ebay.com"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <img className={styles.logo} src={ebay} alt="eBay" />
+          </a>
+          <a
+            href="http://www.srvd.vet"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <img className={styles.srvdLogo} src={srvd} alt="SRVD" />
+          </a>
+          <a
+            href="http://www.oregonpva.org"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <img className={styles.pvaLogo} src={PVA} alt="Oregon PVA" />
           </a>
```

Comment:
I suggest wrapping these `<a>` elements with `<div>` elements that can then be used to constrain the child image sizes and enforce a consistent height/width.  The sizes and layout of of these various images is still somewhat inconsistent. 

For example, the Sabio logo still appears larger than the others, and the CodeFellows one is aligned to the bottom, and the horizontal alignment appears a bit off as well: 
![screenclip](https://user-images.githubusercontent.com/8835055/32312137-751a2ae4-bf72-11e7-8cb4-961fce7c289f.png)